License.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Jonathan Keam
```

Comment:
it is not Jonathan Keam

Plus license should mention that this project uses http://www.paralint.com/projects/notifu/index.html. Not sure what the license is there (it says open source), but it will be nice to give a credit.

Reply:
My apologies, don't know how I missed that! Moot point as this project is deprecated, but the initial impetus of this was that we had a tool that would look at repos that we were using and ensure that those projects had a valid license (https://help.github.com/articles/licensing-a-repository/).  If they did not, they would be flagged by our tool and that's why I opened the issue.

Thanks for taking the time to reply to this issue though!